cmd/podman/create.go

```diff
@@ -148,52 +149,17 @@ func createCmd(c *cli.Context) error {
 	return nil
 }
 
-func parseSecurityOpt(config *cc.CreateConfig, securityOpts []string) error {
-	var (
-		labelOpts []string
-		err       error
-	)
+func loadAppArmor(config *cc.CreateConfig) error {
```

Comment:
Can you add a comment/description to `loadAppArmor()`?

Reply:
One thing I noticed is that the format of logs and errors is a bit inconsistent in this function. Some start capital, some start lower case. Also `AppArmor profile` vs `AppArmor security profile`.